ansible/roles/common/tasks/main.yml

```diff
@@ -50,6 +51,17 @@
     name: supervisor
     enabled: yes
 
+- name: Permit password sshd auth
```

Comment:
I think we should only enable sshd password auth in the `devel` role, not on production images.

Reply:
Ok, makes sense.